protokube/pkg/gossip/mesh/mesh.pb.go

```diff
@@ -170,6 +170,7 @@ func (m *KVStateRecord) MarshalTo(data []byte) (int, error) {
 	return i, nil
 }
 
+//lint:ignore U1000 - Generated by protoc-gen-gogo from mesh.proto
```

Comment:
I'm not sure we can add comments to this file, wont they get wiped the next time we regenerate the file? Maybe theres a setting in the generation that we can add, or maybe we'll need to permanently exclude this file from staticcheck. It might be possible to configure the generator to add a single [file exclusion comment](https://staticcheck.io/docs#file-based-linter-directives)

Reply:
Nice catch with the file exclusion.
The file has been generated almost 3 years ago and there are already some comments like the copyright. If it's ever regenerated, the static check should catch this again.